### PR TITLE
Berry with Return

### LINKED ISSUE #29: Berry with Return
A strawberry that, when collected, will place you into a bubble and return you to a selected location, similarly to cassettes.

Requested by LethargicDoggo in collabcord

### PR DESCRIPTION
**Pull request by WoofWoofDoggo**
A strawberry (also available as a winged strawberry) which when picked up transports the player back to their spawnpoint via bubble (similar to Cassettes and keys.)

Requested by LethargicDoggo and Mun in collabcord

Fixes #29

### CODE REVIEW

**Comment by 0x0ade** [comment]
Uhh... the more I look at your PR, the more it looks like you've accidentally included the changes for your "remove light sources trigger" as well. Please fix this first.

**Comment by WoofWoofDoggo** [comment]
Those changes have been removed (thanks Max 😅)

I think this branch is fine for merging now?

**Comment by 0x0ade** [comment]
Still awaiting a request on the Ahorn plugin from @Cruor - dunno why his review wasn't requested yet.

File: Ahorn/entities/bubbleReturnBerry.jl [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const placements = Ahorn.PlacementDict(
+	"Strawberry With Return (Spring Collab 2020)" => Ahorn.EntityPlacement(
+		ReturnBerry
+	)
+)
+
+function getSpriteName(entity::ReturnBerry)
+	winged = get(entity.data, "winged", false)
+
+	if(winged)
```
Comment (Cruor) [inline Ahorn/entities/bubbleReturnBerry.jl]: This isn't C#, please use `if winged` instead

**Comment by WoofWoofDoggo** [comment]
@Cruor Removed the parentheses, does the Ahorn plugin look fine to you now?

**Pull request merged**